src/meta/processors/partsMan/CreateSpaceProcessor.cpp

```diff
@@ -9,6 +9,7 @@
 
 DEFINE_int32(default_parts_num, 100, "The default number of parts when a space is created");
 DEFINE_int32(default_replica_factor, 1, "The default replica factor when a space is created");
+DEFINE_int32(default_max_parts_num, 100, "The default max partitions number");
```

Comment:
I think you can set default to max, and give it a property value in configuration when deploy.

Reply:
100 partitions is enough for trial version, user can create up to 100 spaces.

Reply:
Currently,  the largest number for parts is 2^24,   the default max number should be it.